src/Apps/Order/Routes/Payment/index.tsx

```diff
@@ -31,46 +61,57 @@ export class PaymentRoute extends Component<PaymentProps> {
 
         <Spacer mb={3} />
 
-        <Responsive>
-          {({ xs }) => (
-            <TwoColumnLayout
-              Content={
-                <>
-                  <Join separator={<Spacer mb={3} />}>
-                    <Placeholder height="68px" name="Credit Card" />
-                    <Placeholder height="20px" name="Billing/Shipping Check" />
-                    {!xs && (
-                      <Link to={`/order2/${order.id}/review`}>
-                        <Button size="large" width="100%">
-                          Continue
-                        </Button>
-                      </Link>
+        <Elements>
+          <Responsive>
+            {({ xs }) => (
+              <TwoColumnLayout
+                Content={
+                  <>
+                    <Join separator={<Spacer mb={3} />}>
+                      <CreditCardInput />
+                      <Checkbox
+                        selected={this.state.hideBillingAddress}
+                        onSelect={hideBillingAddress =>
+                          this.setState({ hideBillingAddress })
+                        }
+                      >
+                        Billing and shipping addresses are the same
+                      </Checkbox>
+                      <Collapse open={!this.state.hideBillingAddress}>
+                        <AddressForm
+                          country={this.state.country}
+                          onUpdateName={this.onUpdateName}
```

Comment:
non-blocking: I hadn't seen this component before, and would like to take the opportunity to +1 ash's suggestion from #1152 to collapse these individual props into a single one.